.gitignore

```diff
@@ -3,4 +3,6 @@
 
 __pycache__/
 *.py[cod]
-*$py.class
\ No newline at end of file
+*$py.class
+
+lonabot.db
```

Comment:
If anything, this should be `*.db`, although it is not relevant to the current PR.